test/production/escheck-output/index.test.ts

```diff
@@ -23,29 +22,4 @@ describe('ES Check .next output', () => {
       'info: ES-Check: there were no ES version matching errors!  🎉'
     )
   })
-
-  it('should emit ES5 with legacyBrowsers: true', async () => {
```

Comment:
This test is fundamentally broken.

x-ref: https://vercel.slack.com/archives/C04DUD7EB1B/p1681534087348619